webui/react/src/routes/index.ts

```diff
@@ -49,10 +52,22 @@ export const appRoutes: RouteConfigItem[] = [
     path: '/det/logout',
     title: 'Logout',
   },
+  {
+    exact: true,
+    id: '',
+    path: '/',
+    redirect: '/det/dashboard',
+  },
+  {
+    id: 'catch-all',
+    path: '*',
+    redirect: '/det/login',
```

Comment:
go to our default route?

Reply:
`+` I think that could save us a redirect based on 
`        if (route.needAuth && !auth.isAuthenticated) {`